client/components/booking/TimeSlotGrid.tsx

```
@@ -0,0 +1,41 @@
+      {timeSlots.map((slot) => (
+        <button
+          key={slot._id}
+          className={cn(
+            "py-2 px-3 rounded-md text-sm font-medium transition-colors",
+            "border hover:bg-primary/10",
+            selectedSlot?._id === slot._id 
+              ? "bg-primary text-primary-foreground hover:bg-primary/90 border-primary" 
+              : "bg-background text-foreground border-input"
+          )}
+          onClick={() => onSelect(slot)}
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)